virtinst/domain/cputune.py

```diff
@@ -16,6 +16,14 @@ class _VCPUPin(XMLBuilder):
     vcpu = XMLProperty("./@vcpu", is_int=True)
     cpuset = XMLProperty("./@cpuset")
 
+class _EmulatorPin(XMLBuilder):
+    """
+    Class for generating <cputune> child <emulatorpin> XML
+    """
+    XML_NAME = "emulatorpin"
+
+    cpuset = XMLProperty("./@cpuset")
+
 
```

Comment:
Whitespace issue here: previously the classes were separated by two blanklines, but now it's 1 line before _EmulatorPin and 3 lines after.

Reply:
FWIW `./setup.py pylint` pep8 call would have caught this too

Reply:
It did. I just derped and forgot to run it *before* committing ;)